integrations/Lotame/browser.js

```diff
@@ -0,0 +1,104 @@
+import logger from "../../utils/logUtil";
+import {LotameStorage} from "./LotameStorage";
+class Lotame {
+  constructor(config) {
+    this.name = "LOTAME";
+    this.storage = LotameStorage;
+    this.bcpUrlSettings = config.bcpUrlSettings;
+    this.dspUrlSettings = config.dspUrlSettings;
+    this.mappings = {};
+    config.mappings.forEach( mapping => {
+      let key = mapping.key;
+      let value = mapping.value;
+      this.mappings[key] = value;
+    });
+  }
+
+  init() {
+    logger.debug("===in init Lotame===");
+    window.LOTAME_SYNCH_CALLBACK = () => {};
+  }
+
+  addPixel(source, width, height){
+    let image = document.createElement('img'); 
+    image.src =  source;
+    image.setAttribute("width", width);
+    image.setAttribute("height", height);
+    document.getElementsByTagName('body')[0].appendChild(image);
+  }
+
+  synchPixel(userId){
+    logger.debug("===== in synchPixel ======");
+
+    if(this.dspUrlSettings && this.dspUrlSettings.length > 0){
+      this.dspUrlSettings.forEach(urlSettings => {
+        let dspUrl = this.compileUrl({...this.mappings, userId:userId}, urlSettings.dspUrlTemplate);
+        this.addPixel(dspUrl, "1", "1");
+      });
+    }
+    this.storage.setLotameSynchTime(Date.now());
+    if(window.LOTAME_SYNCH_CALLBACK && typeof window.LOTAME_SYNCH_CALLBACK == "function"){
+      window.LOTAME_SYNCH_CALLBACK();
+    }
+
+  }
+
+  compileUrl(map, url){
+
+    Object.keys(map).forEach(key => {
+      let replaceKey = "{{"+key+"}}";
+      let regex = new RegExp(replaceKey, 'gi');
+      url = url.replace(regex, map[key]);
+    });
+    console.log(url);
```

Comment:
logger.debug?

Reply:
forgot to remove it. not needed. removing it now.